core/src/test/java/it/pagopa/selfcare/mscore/core/DelegationServiceImplTest.java

```diff
@@ -230,6 +233,29 @@ void find_shouldGetData_fullMode_defaultPage() {
         assertEquals(delegation.getId(), response.get(0).getId());
     }
 
+    /**
+     * Method under test: {@link DelegationServiceImpl#getDelegationsV2(GetDelegationParameters)}
+     */
+    @Test
+    void getDelegationsV2_shouldGetData() {
+        //Given
+        Delegation delegation = new Delegation();
+        delegation.setId("id");
+        DelegationWithPagination delegationWithPagination = new DelegationWithPagination(List.of(delegation), new PageInfo(10, 0, 10, 1));
+        when(delegationConnector.findAndCount(any())).thenReturn(delegationWithPagination);
+        //When
+        DelegationWithPagination response = delegationServiceImpl.getDelegationsV2(createDelegationParameters("from", "to", "productId", null, null,
+                GetDelegationsMode.NORMAL, null, 0, 100));
+        //Then
+        verify(delegationConnector).findAndCount(any());
```

Comment:
you could use ArgumentCaptor for checking if findAndCount use really params present in createDelegationParameters(...)...you can find some examples in this repo